dashboard/app/controllers/registrations_controller.rb

```diff
@@ -542,6 +582,32 @@ def edit
       params[:user][:password].present?
   end
 
+  private def new_user_permitted_params
```

Comment:
I think this would actually be a great candidate to move into [Policies](https://github.com/code-dot-org/code-dot-org/tree/staging/dashboard/lib/policies). There isn't a `registration` policy yet, but the desire is to move stuff like this, that doesn't _need_ to exist in a controller or model, into Policies. This helps us modularize things and mitigate controller and model bloat.

Checkout how we've implemented something very similar in the [lti policy](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/lib/policies/lti.rb#L50).

Then from the context of this controller, to you would just do something like `Policies::Registration::NEW_USER_PERMITTED_PARAMS` if this were a variable, or `Policies::Registration.new_user_permitted_params` if it were a method.